src/Engine/ProtoCore/DSASM/InstructionSet.cs

```diff
@@ -113,7 +113,7 @@ public struct MetaData
     /// <summary>
     /// Wrapper over all values handled by the MSIL/CLR engine.
     /// </summary>
-    internal struct CLRStackValue
+    public struct CLRStackValue
```

Comment:
We should keep everything as `internal` and just expose internals to other assemblies. Don't want end users to access any crap from `ProtoCore`, etc.

Reply:
see comment below